docs/RELEASE_PROCESS.md

```diff
@@ -0,0 +1,11 @@
+### `cosmos/cosmos-sdk` Release Process
+
+- [ ] 1. Decide on release designation (are we doing a patch, or minor version bump) and start a P.R. for the release
+- [ ] 2. Add commits/PRs that are desired for this release **that haven’t already been added to develop**
```

Comment:
What might these be? Do you mean updating `version/version.go`?

Reply:
I don't know. This addition was at the request of @jaekwon

Reply:
Hmm OK, let's clarify.

Reply:
@jaekwon can you clarify the above? ☝️

Reply:
I think we want to keep this as optional for stuff like `version/version.go` or `CHANGELOG.md` that might need fixing in the release branch?

Reply:
I'm thinking that we merge PRs into develop during our normal develop process, then when it comes time to release, we create a release meta-PR forked off of develop that further merges remaining PRs.  Maybe this isn't what we should be doing?

Reply:
I guess I just don't understand what the "remaining PRs" would be - why would we merge a PR to a release branch if we hadn't already merged and tested it on develop?